workflow.schema.json

```diff
@@ -0,0 +1,531 @@
+                                            "description": "Unique identifier of the organization integration"
+                                        }
+                                    },
+                                    "$source": "orgintegrations._id",
+                                    "required": [
+                                        "$oid"
+                                    ]
+                                },
+                                "action_type": {
+                                    "type": "string",
+                                    "description": "Type of the action",
+                                    "$source": "integrationactions.actionType"
+                                },
+                                "action_name": {
+                                    "type": "string",
+                                    "description": "Name of the action",
+                                    "$source": "integrationactions.name"
+                                },
+                                "action_description": {
+                                    "type": "string",
+                                    "description": "Description of the action",
+                                    "$source": "integrationactions.description"
+                                },
+                                "break_on_error": {
+                                    "type": "boolean",
+                                    "description": "Indicates if the action should break on error"
+                                },
+                                "action_data": {
+                                    "type": "array",
+                                    "items": {
+                                        "type": "object",
+                                        "$comment": "As many as users we are applying the action to",
+                                        "properties": {
+                                            "k": {
+                                                "type": "string",
+                                                "$comment": "Object ID referencing user collection/ID"
+                                            },
+                                            "v": {
+                                                "type": "object",
+                                                "$comment": "To some extent derived from action in Mongo DB. Inputs to the action to transform into the external API call"
+                                            }
+                                        }
+                                    }
+                                },
+                                "manual_action_template_id": {
+                                    "type": "string",
+                                    "description": "Identifier of the manual action template"
+                                }
+                            },
+                            "required": [
```

Comment:
Is `action_id` required?

Reply:
Same for `org_integration_id`

Reply:
Hi @deipkumar from my tests using internal validation API, looks like these two are required